### DISCUSSION TITLE
Can I use my Github page as a blog on free software matters?

### DISCUSSION THREAD

**Discussion by ineuw** [community/community#53059] [New to GitHub] [answered]
I would like to use the Discussion section of my page to comment and discuss experiences with Linux, and free software from a user's point of view.

I am not a coder only a dedicated user, and I don't plan to post code, just open discussions.

Is this permitted?

**Comment by kvten** [comment, marked as answer]
Doesn't seem to be against terms of service.

https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#c-acceptable-use

**Comment by ineuw** [comment]
Thanks for the link.
